ambry-cloud/src/test/java/com.github.ambry.cloud/azure/AzureIntegrationTest.java

```diff
@@ -119,9 +126,17 @@ public void testNormalFlow() throws Exception {
     metadata = azureDest.getBlobMetadata(Collections.singletonList(blobId)).get(blobId.getID());
     assertEquals(deletionTime, metadata.getDeletionTime());
     assertEquals(azureDest.getAzureBlobName(blobId), metadata.getCloudBlobName());
+
     azureDest.purgeBlob(metadata);
     assertTrue("Expected empty set after purge",
         azureDest.getBlobMetadata(Collections.singletonList(blobId)).isEmpty());
+
+    // Get blob should fail after purge
+    try {
+      verifyDownloadMatches(blobId, uploadData);
+    } catch (CloudStorageException csex) {
+      assertTrue(csex.getMessage().contains("Error downloading blob"));
```

Comment:
Minor: not good practice for tests to check for exact error text, makes tests brittle.